docs/references/databases/update-relationship-attribute.md

```diff
@@ -0,0 +1 @@
+Update relationship attribute.
```

Comment:
```suggestion
Update relationship attribute. [Learn more about relationship attributes](docs/databases-relationships#relationship-attributes).
```

Reply:
Practically speaking, does this update the attribute, or does it basically update the relationship?

Reply:
I would say both; the relationship **is** the attribute